### PR TITLE
Fix: Boss General button positions on General's Power screen

### PR DESCRIPTION
**Pull request by commy2**
### default positions from 1.04

shot_20230826_011418_2

shot_20230826_011558_3

- Pathfinders, Cluster Mines require rank 3, but are listed under rank 1.
- Carpet Bomber is listed under rank 3, but requires rank 5.

---

### intermediate change

shot_20230826_012152_1

- powers moved under their appropriate required rank

---

### this change complete

shot_20230826_013400_1

- position of fourth rank 5 power fixed, no other general uses a fourth rank 5 power.

### CODE REVIEW

File: Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+date: 2023-08-26
+
+title: Fixes position of General's Powers on Boss General promotion screen.
+
+changes:
+  - fix: Move Pathfinder unlock below rank 3.
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml]: I suggest to describe as "Move X unlock to Rank 3 section of Promotion screen."

File: Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+date: 2023-08-26
+
+title: Fixes position of General's Powers on Boss General promotion screen.
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml]: Remove dot it title.

File: Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+changes:
+  - fix: Move Pathfinder unlock below rank 3.
+  - fix: Move Cluster Mines unlock below rank 3.
+  - fix: Move Carpet Bomber General's Power below rank 5.
+  - fix: UI Position of fourth rank 5 General's Power.
+
+labels:
+  - boss
+  - fix
+  - minor
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml:16]: Add gui label.

File: Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  - v1.0
+
+links:
+  - https://github.com/TheSuperHackers/GeneralsGamePatch/pull/xxxx
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml]: xxxx

**Review by xezon — APPROVED** [review]
Very Nice.

File: Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml (hunk 1 above)
Comment (commy2) [reply Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml]: I don't want to confuse people and make them assume it was previously a Rank 1 unlock etc.

Comment (xezon) [reply Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml]: Ok try:

"Move the Pathfinder unlock button to the correct rank section in the Promotion screen."

**Comment by commy2** [comment]
I changed my mind and moved Cluster Mines into its own column.

shot_20230826_094826_1

File: Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml (hunk 3 above)
Comment (commy2) [reply Patch104pZH/Design/Changes/v1.0/xxxx_boss_promotion_screen.yaml:16]: done

**Pull request merged**